Platform/ApollolakeBoardPkg/BoardConfig.py

```diff
@@ -198,7 +198,7 @@ def GetPlatformDsc (self):
             'SpiFlashLib|Silicon/CommonSocPkg/Library/SpiFlashLib/SpiFlashLib.inf',
             'IgdOpRegionLib|Silicon/$(SILICON_PKG_NAME)/Library/IgdOpRegionLib/IgdOpRegionLib.inf',
             'IocIpcLib|Platform/$(BOARD_PKG_NAME)/Library/IocIpcLib/IocIpcLib.inf',
-            'BootGuardLib|Silicon/$(SILICON_PKG_NAME)/Library/BootGuardLib20/BootGuardLib20.inf',
+            'BootGuardLib|Silicon/CommonSocPkg/Library/BootGuardLib20/BootGuardLib20.inf',
```

Comment:
Why this moved to common? It should be part of Applilolake.

Reply:
I moved it to common because it is not just used by APL -- it is also used by QEMU.

Reply:
Made QEMU consume the BtG library already in common and moved this BtG library back to APL.